platforms/posix/src/px4/common/px4_daemon/pxh.cpp

```diff
@@ -551,4 +555,18 @@ void Pxh::_tab_completion(std::string &mystr)
 	}
 }
 
+int Pxh::shutdown_handler(int argc, char *argv[])
+{
+	printf("Force shutting down...\n");
+	fflush(stdout);
+
+	// Force kill all child processes
+	int ret = system("pkill -9 -f gz");
+	(void)ret;  // Suppress warning about unused return value
+	printf("Killed gz...\n");
+	ret = system("pkill -9 -f px4");
```

Comment:
This seems wrong, why should px4 kill itself with pkill? Is there a reason we can't shutdown cleanly in the first place?